src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -127,6 +128,7 @@ def get_warning_args_default(cls):
             "-C-Xlint:all",
             "-C-Xlint:-serial",
             "-C-Xlint:-path",
+            "-C-Xlint:unchecked",
```

Comment:
This is a surprising change

Reply:
This was pushed by mistake. Will remove.